src/assets/pos.ts

```diff
@@ -41,19 +39,26 @@ import {
 	Staker,
 	Stake,
 	PoSStoreEntry,
+	StakerBuffer,
 } from '../types';
-import { genesisPoSSchema } from '../schemas';
+
+const ceiling = (a: number, b: number) => {
+	if (b === 0) throw new Error('Can not divide by 0.');
+	return Math.floor((a + b - 1) / b);
```

Comment:
though not needed for current usage, but could we use `Math.ceil` instead? This won't work incase b <= 1. Ex: a=5.5, b=1. 
If we can not use `Math.ceil`, then maybe we can substract a smaller value like 1e-4?

Reply:
The implementation is as per the LIP specifications and specific to the Lisk Core migration. Kindly do not modify!